bindings/python/crocoddyl/multibody/costs/contact-cop-position.cpp

```diff
@@ -16,14 +16,16 @@ namespace python {
 void exposeCostContactCoPPosition() {
   bp::class_<CostModelContactCoPPosition, bp::bases<CostModelAbstract> >(
       "CostModelContactCoPPosition",
-      bp::init<boost::shared_ptr<StateMultibody>, boost::shared_ptr<ActivationModelAbstract>, FrameFootGeometry, 
-      const Eigen::Ref<const Eigen::VectorXd>&>(
-          bp::args("self", "state", "activation", "foot_geom", "normal"),
+      bp::init<boost::shared_ptr<StateMultibody>, boost::shared_ptr<ActivationModelAbstract>, 
+      const std::size_t&, const Eigen::Ref<const Eigen::VectorXd>&, const Eigen::Ref<const Eigen::VectorXd>&, int>(
```

Comment:
1. Use Vector2 and Vector3, instead of VectorXd.
2. Do not use `Eigen::Ref`

Reply:
Shall we directly access the typedef `Vector2s&`, or what do you suggest instead?